fog/view/connection/src/fog_view_router_client.rs

```diff
@@ -197,6 +197,12 @@ impl FogViewRouterGrpcClient {
     }
 }
 
+impl Drop for FogViewRouterGrpcClient {
+    fn drop(&mut self) {
+        block_on(self.request_sender.close()).expect("Couldn't close the router request sender");
```

Comment:
:clown_face: 
Not sure if this is better.  Previously an error message was happening when this wasn't closing, but it's not clear if grpcio is in a bad state or if it can recover.  If it can recover, it seems less than ideal to abort our application on a failure to close.
```suggestion
        let _ = block_on(self.request_sender.close());
```

Reply:
Using `expect` when shutting down `grpcio` servers or connections is common across our codebase so will keep it as is.